hathor/p2p/utils.py

```diff
@@ -200,3 +202,28 @@ def format_address(addr: IAddress) -> str:
         return f'{host}:{port}'
     else:
         return str(addr)
+
+
+def to_height_info(raw: tuple[int, str]) -> HeightInfo:
+    """ Instantiate HeightInfo from a literal tuple.
+    """
+    if not (isinstance(raw, list) and len(raw) == 2):
+        raise ValueError(f"height_info_raw must be a tuple with length 2. We got {raw}.")
+
+    height, id = raw
+
+    if not isinstance(id, str):
+        raise ValueError(f"id (hash) must be a string. We got {id}.")
+    hash_pattern = r'[a-fA-F\d]{64}'
+    if not re.match(hash_pattern, id):
+        raise ValueError(f"id (hash) must be valid. We got {id}.")
+    if not isinstance(height, int):
+        raise ValueError(f"height must be an integer. We got {height}.")
+    if height < 0:
+        raise ValueError(f"height must be greater than or equal to 0. We got {height}.")
+
+    return HeightInfo(height, bytes.fromhex(id))
+
+
+def to_serializable_best_blockchain(best_blockchain: list[HeightInfo]) -> list[tuple[int, str]]:
```

Comment:
Well, it would not change much, right? Because the result of this function will pass though a json_dumps anyway in the ready state, and also will pass through a formatter in the resource response. I believe we can keep this function and the HeightInfo as it is. Besides this two points, the ready state and status resource, HeightInfo do not need to be converted to json.

Reply:
Ok.